GTMAppAuthSwift/Sources/AuthStateStore.swift

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/// Represents the operations needed to provide credential storage.
+@objc(GTMAuthStateStore)
+public protocol AuthStateStore {
+  /// The `String` name for the stored credential.
+  @objc var credentialItemName: String { get }
```

Comment:
We should move away from the `crendential` naming to avoid confusion.  How about `authStateItemName`?

Reply:
What about `itemName` since I think it's obvious that the item refers to auth state?

Reply:
That works.

Reply:
Sounds good. Then change made!